sample/ApiSamples/Program.cs

```diff
@@ -28,18 +28,18 @@ private static void Main(string[] args)
 				}
 
 				Thread.Sleep(1000);
-				Console.WriteLine("Continue trace finished");
+				Console.WriteLine("Callee side finished");
 			}
 			else
 			{
-				Console.WriteLine("Start");
+				Console.WriteLine("Started caller side");
 				PassTraceContext();
 
 				//WIP: if the process terminates the agent
 				//potentially does not have time to send the transaction to the server.
 				Thread.Sleep(1000);
 
-				Console.WriteLine("Done");
+				Console.WriteLine("Caller side is about to exit - press any key...");
```

Comment:
small nit: I usually just replace the call to `PassTraceContext();` to whatever sample method I need to test, and with that there isn't really a caller/callee side, because the rest of the sample methods only use 1 process, so this text would be misleading. I plan to add some kind of menu to this app that lists all the public sample methods and then you can enter e.g. a number to run a specific one.

Reply:
I can rename "Caller" to "Process #1" and "Callee" to "Process #2". Will it work?

Reply:
Yeah, I think that would be better. Or maybe use the process id with `Process.GetCurrentProcess().Id`.

Reply:
Done